docs/src/main/asciidoc/user-guide/topics/import_files_to_panel.adoc

```diff
@@ -0,0 +1,284 @@
+[id='import-files-to-panel']
+= Import XML / JSON Schema, Instance or Java Class Files
+
+Data mapping fundamentally involes morphing one data shape into another.  A user
+defines these shapes using XML schemas, JSON notation or by establishing a simple Java
+class.  Lets examine each file type independently.
+
+* JSON Schema File: *JSONSchema.json*
+
+```
+  {
+    "$schema": "http://json-schema.org/schema#",
+    "description": "Order",
+    "type": "object",
+    "properties": {
+      "order": {
+        "type": "object",
+        "properties": {
+          "address": {
+          "type": "object",
+          "properties": {
+            "street": { "type": "string" },
+            "city": { "type": "string" },
+            "state": { "type": "string" },
+            "zip": { "type": "string" }
+          }
+        },
+        "contact": {
+          "type": "object",
+          "properties": {
+            "firstName": { "type": "string" },
+            "lastName": { "type": "string" },
+            "phone": { "type": "string" }
+          }
+        },
+        "orderId": { "type": "string" }
+      }
+    },
+    "primitives": {
+      "type": "object",
+      "properties": {
+        "stringPrimitive": { "type": "string" },
+        "booleanPrimitive": { "type": "boolean" },
+        "numberPrimitive": { "type": "number" }
+      }
+    },
+    "primitiveArrays": {
+      "type": "object",
+      "properties": {
+        "stringArray": {
+          "type": "array",
+          "items": { "type": "string" }
+        },
+        "booleanArray": {
+          "type": "array",
+          "items": { "type": "boolean" }
+        },
+        "numberArray": {
+          "type": "array",
+          "items": { "type": "number" }
+        }
+      }
+    },
+    "addressList": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "street": { "type": "string" },
+          "city": { "type": "string" },
+          "state": { "type": "string" },
+          "zip": { "type": "string" }
+        }
+      }
+    }
+    }
+  }
+```
+Lets say we'd like to import this file into the AtlasMap *Source* panel.  Simply select the import icon
```

Comment:
Instead: To import this file into the AtlasMap *Source* panel, select the import icon at the top of the panel.